meson.build

```diff
@@ -2711,19 +2717,20 @@ errnos_h = custom_target('errnos.h',
   command : ['tools/all_errnos', cc.cmd_array()]
 )
 
-mq_libs = []
-mq_libs += cc.find_library('rt', required : true)
+lib_rt = cc.find_library('rt', required : get_option('build-lsfd'))
```

Comment:
This fails if librt is not present but also not needed. (Same as in current master)
Could you change this to reuse the results from the checks above?

Reply:
Because it doesn't check for it all because it is using an option here. Got it.

Reply:
Alright, I have the `cc.find_library` call happen in one place now instead of three. I used a `require` call on the `build-lsfd` feature option to require the rt library be found.